### DISCUSSION TITLE
Coverart Fetch Not Working

### DISCUSSION THREAD

**Discussion by jackjhill** [rern/rAudio#662] [General]
I still cannot get rAudio 1 to fetch missing cover art like it used to - hasn't been working for me for a couple of months.  Am on latest version.

Any ideas what I can check to see what has happened?

Regards

JJH

**Comment by rern** [comment]
Try:
`sed -i 's|\(\s*coverart=/data/shm\)\(.*\)|\1/webradio\2|' /srv/http/bash/status.sh`

**Comment by jackjhill** [comment]
Just tried that rern (from an ssh session, logged in as root).  No difference when I navigate to folder without album art.

The think I can't understand is that it was working ok a while back, I fiddled with some settings on account of some other issue - which I can't recall), and ever since has stopped working.

Folders are hosted on a truenas server, but permissions are not an issue as I can do the long click method and get art from my desktop.

Not the end of the world but it is bugging me as it used to work ok.

JJH

**Comment by rern** [comment]
The bug was found together with replace and remove. To be fixed with next update.

**Comment by jackjhill** [comment]
Thanks for the attention rern.

JJH
